src/dmd/root/filename.d

```diff
@@ -631,71 +631,74 @@ nothrow:
         }
     }
 
+    /**
+       Ensure that the provided path exists
+
+       Accepts a path to either a file or a directory.
+       In the former case, the `basepath` (path to the containing directory)
+       will be checked for existence, and created if it does not exists.
+       In the later case, the directory pointed to will be checked for existence
+       and created if needed.
+
+       Params:
+         path = a path to a file or a directory
+
+       Returns:
+         `true` if the directory exists or was successfully created,
+         `false` otherwise
```

Comment:
This second line is redundant.

Reply:
Technically it could throw which would mean it wouldn't return at all.